src/Models/SeoScan.php

```diff
@@ -40,4 +43,13 @@ public function scores(): HasMany
     {
         return $this->hasMany(SeoScore::class);
     }
+
+    public function prunable(): Builder
+    {
+        if (! config('seo.database.prune.older_than_days')) {
```

Comment:
Is deze niet gevaarlijk dat die dan de hele table leegt? @Baspa

Reply:
Yes je had gelijk